aspnetcore/blazor/file-uploads.md

```diff
@@ -101,26 +101,88 @@ If you're using the [Autofac Inversion of Control (IoC) container](https://autof
 
 ## File size read and upload limits
 
-:::moniker range=">= aspnetcore-6.0"
+:::moniker range=">= aspnetcore-9.0"
+
+Server-side or client-side, there's no file read or upload size limit for the <xref:Microsoft.AspNetCore.Components.Forms.InputFile> component.
```

Comment:
This statement sounds like it contradicts the earlier statement in this doc that "Reading one file or multiple files larger than 500 KB results in an exception." Should this statement just be removed?

```suggestion
```

Reply:
It just means that there's no restriction from the component itself ... that it doesn't try to set any such limit. This is stated to make it clear that any limits aren't due to the use of the component. Do you still want to strike it, or perhaps it should be rephrased to something like ...

```suggestion
If any file size upload limits are imposed during a file upload, they aren't imposed by the <xref:Microsoft.AspNetCore.Components.Forms.InputFile> component, which never sets an upload limit server-side or client-side.
```

Reply:
I think we should just remove the statement and focus the content on the limits that are actually relevant.